ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -571,18 +573,25 @@ private NamedBlobRecord run_get_v2(String accountName, String containerName, Str
           return new NamedBlobRecord(accountName, containerName, blobName, blobId, timestampToMs(deletionTime),
               version);
         }
+      } catch (Exception e) {
```

Comment:
try-with-resource statement can actually have a catch clause, something like
```
try (ResultSet resultSet = statement.executeQuery() ) {
} catch (Exception e) {
    logger.error("xxxx");
    throw e;
}

Reply:
Sure. Added back the try-with-resource statement